src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -315,6 +315,14 @@ private function fixSpaceAboveClassElement(Tokens $tokens, array $class, int $el
         $this->correctLineBreaks($tokens, $nonWhiteAbove, $element['start'], $this->determineRequiredLineCount($tokens, $class, $elementIndex));
     }
 
+    /**
+     * @param array{
+     *     index: int,
+     *     open: int,
+     *     close: int,
+     *     elements: non-empty-list<array{token: Token, type: string, index: int, start: int, end: int}>
+     * } $class
+     */
```

Comment:
3 usages in the same file, so it's better to define class-wide alias (@keradus started convention with underscore prefix, so `_Class` in this case).